---
camera_topics:
 - camera1
 - /192_168_10_241/artags/artag1/ar_pose_marker


